app/views/sign_in_tokens/email_not_recognised.html.erb

```diff
@@ -14,12 +14,8 @@
     </p>
     <p class="govuk-body">
       If you think your email address is correct and you should have access,
-      you can email our support team:
+      you can <%= govuk_link_to "contact our support team", support_ticket_path %>.
     </p>
-    <ul class="govuk-list">
-      <li>Email: <%= ghwt_contact_mailto %></li>
-      <li>We aim to respond within 4 working days.</li>
-    </ul>
```

Comment:
Removed the reference to the SLA because its mentioned on the get support start page